### DISCUSSION TITLE
"Using 8 of 4 threads" thread numbering again?

### DISCUSSION THREAD

**Discussion by astesin** [mydumper/mydumper#1036] [Q&A]
myloader was started with -t 6

2023-02-15 14:53:44 [INFO] - Thread 3: restoring `mydb`.`mytable` part 5 of 8 from mydb.mytable.00000.sql. Progress 12 of 1205. Using **5 of 4** threads.
2023-02-15 14:53:55 [INFO] - Thread 1: restoring `mydb`.`myothertable` part 1 of 8 from mydb.myothertable.00000.sql. Progress 13 of 1205. Using **8 of 4** threads.

Hi, I suspect it is a harmless funny behavior, but looks somehow frightening :) Thanks a lot for your attention :)

BTW should I update from the 0.13.1-2 to a more recent version? 0.13.1-3, or 0.13.2, or maybe 14.smth? Warmest regards, Andreas

**Comment by davidducos** [comment]
Hi @astesin,
You should test with latest prerelease. I think that I already fixes that thread numbering issue, if not, I'm sure that is working with the numbers that it should.
Best

**Reply by astesin** [reply]
Is v0.14.0-3 Ok to test? I'm not ready to dig inside the job of building from sources, sorry, please accept my apologies. Or I'll wait until the next bunch of .deb packages will become available.

**Reply by davidducos** [reply]
@astesin please, review the releases section and you will find new releases/prereleases
